lib/js/components/buttons/IconButton/IconButton.vue

```diff
@@ -282,7 +282,7 @@ export default {
 			throw new Error(`Color: ${this.color} is supported only in type: "icon-only"`);
 		},
 		isButtonColor(): boolean {
```

Comment:
From what I see, all available colors for IconButton are defined for Button. IMO, we can remove this function.
The same comment applies to setting color directly in `style` attribute in this component